Plugins/Public/base_plugin/Main.cpp

```diff
@@ -976,6 +987,37 @@ bool UserCmd_Process(uint client, const wstring &args)
 		PlayerCommands::BaseLstAllyTag(client, args);
 		return true;
 	}
+	else if (args.find(L"/base addfac") == 0)
+	{
+		returncode = SKIPPLUGINS_NOFUNCTIONCALL;
+		PrintUserCmdText(client, L"Proceeding...");
```

Comment:
???

Reply:
It looks normal, I see nothing unusual. Perhaps do you mean not necessary console msg? Removed.